<ui version="4.0" >
 <class>MemberList</class>
 <widget class="QWidget" name="MemberList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="0" colspan="4" >
    <widget class="QTreeView" name="treeView" />
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="RecvMediaButton" >
     <property name="text" >
      <string>接收</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="sendButton" >
     <property name="text" >
      <string>邀请</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>所在会议：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QLabel" name="confName" >
     <property name="text" >
      <string>未进入会议</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="ExitConfButton" >
     <property name="text" >
      <string>退出会议</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="handButton" >
     <property name="text" >
      <string>举手</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="videoControlButton" >
     <property name="text" >
      <string>停止视频发送 </string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="audioControlButton" >
     <property name="text" >
      <string>停止音频发送 </string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
